asmcomp/i386/NOTES.md

```diff
@@ -18,5 +18,3 @@ Debian architecture name: `i386`
 * ELF application binary interface:
   _System V Application Binary Interface,
    Intel386 Architecture Processor Supplement_
-* MacOS X application binary interface:
-  _OS X ABI Function Call Guide: IA-32 Function Calling Conventions_
```

Comment:
Just for the record, could you please add the following to this `NOTES.md` file?  (I would have edited it myself but I don't have permissions, something that should not happen IMO.)
```
In OCaml versions 3.09.2 to 4.08, MacOS was supported by this port.  Support was removed in OCaml 4.09.
```

Reply:
Done